tools/tsconfig/base.json

```diff
@@ -9,6 +9,7 @@
     "forceConsistentCasingInFileNames": true,
     "inlineSources": false,
     "isolatedModules": true,
+    "module": "ESNext",
```

Comment:
is there a risk that this breaks common js usage of the ai sdk?

Reply:
needed for the rsc move?

Reply:
i think this was required for one of the builds to work when moving to the react package. This just affects how typescript compiles to JS - but we pass it through tsup so it converted at later stage back to commonjs

Reply:
it should have no effect - but i'm going to thoroughly inspect the compiled package to ensure this next